docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
```

Comment:
Quickstart sidenav is probably the one we want for reference docs. (I ended up removing the nav line for python for the time being.)